### PR TITLE
Improve german translation

### PR DESCRIPTION
**Pull request by xdevs23**
Improved the translation and corrected typo.

### CODE REVIEW

File: gui/theme/common/languages/de.xml [hunk 1]
```diff
@@ -186,15 +186,15 @@
...
+		<string name="change_fs_warn2">Dateisysteme nicht. Bitte nochmals überprüfen!</string>
 		<string name="change_fs_confirm">%tw_partition_name% ändern?</string>
 		<string name="formatting">Formatierung...</string>
 		<string name="format_complete">Formatieren abgeschlossen</string>
 		<string name="swipe_change_fs">Dateisystem ändern</string>
 		<string name="swipe_change_s">Ändern</string>
 		<string name="back_btn">Zurück</string>
 		<string name="wipe_enc_btn">Verschlüsselung entfernen</string>
-		<string name="swipe_factory_reset">Werkseinstellungen herstellen</string>
+		<string name="swipe_factory_reset">Zurücksetzen</string>
```
Comment (tyll) [inline gui/theme/common/languages/de.xml:197]: Should this be `Auf Werkseinstellungen zurücksetzen` to match line 135?

File: gui/theme/common/languages/de.xml [hunk 2]
```diff
@@ -119,25 +119,25 @@
...
-		<string name="factory_reset_complete">Werkseinstellungen wiederhergestellt</string>
+		<string name="factory_reset_complete">Auf Werkseinstellungen zurückgesetzt</string>
 		<string name="sel_part_hdr">Partitionen auswählen</string>
-		<string name="wipe_sel_confirm">Ausgewählte Partition(en) löschen?</string>
-		<string name="wiping_part">Lösche Partition(en)...</string>
-		<string name="wipe_complete">Löschen abgeschlossen </string>
-		<string name="sel_part_wipe">Partitionen zum Löschen auswählen:</string>
+		<string name="wipe_sel_confirm">Ausgewählte Partition(en) formatieren?</string>
+		<string name="wiping_part">Formatiere Partition(en)...</string>
+		<string name="wipe_complete">Formatierung abgeschlossen </string>
```
Comment (tyll) [inline gui/theme/common/languages/de.xml:139]: I am not sure, but is `Formatieren abgeschlossen` maybe better to match the next line?

**Review by tyll — COMMENTED** [review]
These look like good changes, there are just two remarks from me.

File: gui/theme/common/languages/de.xml (hunk 2 above)
Comment (xdevs23) [reply gui/theme/common/languages/de.xml:139]: I guess both are ok.

File: gui/theme/common/languages/de.xml (hunk 1 above)
Comment (xdevs23) [reply gui/theme/common/languages/de.xml:197]: Depends on how much space you actually have. If `Auf Werkseinstellungen zurücksetzen` does not get truncated, that would be better.

**Comment by bigbiff** [comment]
Any patches need to be submitted to android_bootable_recovery at https://gerrit.omnirom.org for inclusion.

**Pull request closed without merging**